src/mina-signer/tests/verify-in-snark.unit-test.ts

```diff
@@ -0,0 +1,49 @@
+import { Field, isReady, shutdown } from '../../snarky.js';
```

Comment:
this file tests that the new signatures are really compatible between mina-signer and snarkyjs, and also checks that they can be verified inside a SNARK